apps/portals/src/configurations/adknowledgeportal/synapseConfigs/experimental_tools.ts

```diff
@@ -118,7 +122,7 @@ const experimentalTools: SynapseConfig = {
     shouldDeepLink: true,
     hideDownload: true,
     tableConfiguration: experimentalDetailsTableConfiguration,
-    facetsToFilter: [
+    availableFacets: [
```

Comment:
Accomodate breaking change (in the same PR as the breaking change!)

Reply:
Well that's convenient!